### PR TITLE
Update AWS EBS CSI Driver KEP

### PR DESCRIPTION
**Pull request by leakingtapan**

### CODE REVIEW

**Comment by idealhack** [comment]
KEPs are maintained in k/enhancements now. This PR should be opened there :)

File: keps/sig-aws/20181127-aws-ebs-csi-driver.md [hunk 1]
```diff
@@ -54,13 +56,16 @@ List of driver features include volume creation/deletion, volume attach/detach,
 ### User Stories
 
 #### Static Provisioning
-Operator creates a pre-created EBS volume on AWS and a PV that refer the EBS volume on cluster. Developer creates PVC and a Pod that uses the PVC. Then developer deploys the Pod during which time the PV will be attached to container inside Pod after PVC bonds to PV successfully.
+Operator creates a pre-created EBS volume on AWS and a CSI PV that refers the EBS volume on cluster. Developer creates PVC and a Pod that uses the PVC. Then developer deploys the Pod during which time the PV will be attached to container inside Pod after PVC bonds to PV successfully.
+
+#### Dynamic Provisioning
+Operator creates a storage class that defines EBS CSI drivere as provisioner. Developer creates PVC and a Pod that uses the PVC. A new CSI PV will be created dynamically and be bond to the defined PVC. Finally, the PV will be attached to container inside Pod.
```
Comment (jsafrane) [inline keps/sig-aws/20181127-aws-ebs-csi-driver.md]: drivere -> driver

Comment (jsafrane) [reply keps/sig-aws/20181127-aws-ebs-csi-driver.md]: bond -> bound

**Comment by justaugustus** [comment]
As @idealhack [mentioned](https://github.com/kubernetes/community/pull/3004#issuecomment-443609897), KEPs have moved to [k/enhancements](https://github.com/kubernetes/enhancements).

Please PR any KEP updates there.
/close

**Comment by leakingtapan** [comment]
@jsafrane I created a new PR on enhancement repo [here](https://github.com/kubernetes/enhancements/pull/644) per @justaugustus's request. Your comments are addressed also. Thx

**Pull request closed without merging**